cpp/vm/evmzero/interpreter_test.cc

```diff
@@ -4348,11 +4349,35 @@ TEST(InterpreterTest, JUMPDEST_OutOfGas) {
   });
 }
 
+///////////////////////////////////////////////////////////
+// PUSH0
+TEST(InterpreterTest, PUSH0) {
+  RunInterpreterTest({
+      .code = {op::PUSH0},
+      .state_after = RunState::kDone,
+      .gas_before = 10,
+      .gas_after = 8,
+      .stack_before = {},
+      .stack_after = {0x0},
+      .revision = EVMC_SHANGHAI,
```

Comment:
@HerbertJordan This test checks push0 for Shangai, the rest of the test has baseline Istanbul.